### ISSUE TITLE
Continuity incompatibility

### ISSUE THREAD

**Issue by Nicodemus111** [team-abnormals/blueprint#243]
I am opening another issue for this because when it was reported to sinytra they said it could not be fixed on their side. [https://github.com/Sinytra/Connector/issues/1001](url)
Now that you know the cause is this something that can be fixed?

**Comment by Nicodemus111** [comment]
This issue also happens with athena, it is unrelated to sinytra its just that continuity is not on forge natively.

**Comment by ghost** [comment]
https://github.com/Sinytra/Connector/issues/1001
> This appears to be a conflict between the two mods. if I recall correctly, blueprint replaces minecraft's resource manager with its own, as well as making several other changes to the model system, which eventually cause continuity to stop working. Unfortunately, this cannot be fixed on our side.

Already posted similar issue https://github.com/team-abnormals/blueprint/issues/240 a while ago, but it was closed

**Comment by Nicodemus111** [comment]
Yes, I opened this one and checked sinytra because I saw yours and followed the trail.

**Comment by Liamgrayskull** [comment]
Is reproduceable on my end as well

**Comment by vinnyg0621** [comment]
yes, please fix it! this mod is required for other mods that I have

**Comment by lSherozl** [comment]
For me it causes a crash when the mods are together

**Comment by vinnyg0621** [comment]
when is this stupid incompatibility gonna be fixed? 😤

**Comment by raffle2** [comment]
Can confirm it is also happening to me on forge 1.20.1

**Comment by QianFuv** [comment]
Can confirm

**Comment by QianFuv** [comment]
https://www.curseforge.com/minecraft/mc-mods/blueprint-texture-issue-fix
fix mod here

**Comment by squigglezworth** [comment]
It would be nice if this got fixed.

**Comment by BrettPlays** [comment]
> https://www.curseforge.com/minecraft/mc-mods/blueprint-texture-issue-fix fix mod here

This did not seem to work and appears to still be an issue.

**Comment by XxInvictus** [comment]
Can confirm, was able to completely replicate this on 1.20.1 with only these base mods, issue goes away as soon as Compat O' Plenty and Blueprint are disabled. Can also confirm it is fixed by Blueprint Texture Issue Fix.
- Biomes O' Plenty - 19.0.0.96
- Blueprint - 7.1.3
- Compat O' Plenty - 1.20.1-1.0.0
- Continuity - 3.0.0
- Forgified Fabric API - 0.92.6
- Glitchcore - 0.0.1.1
- Sinytra Connector - 1.0.0-beta46
- Terrablender - 3.0.1.10

**Comment by Noviian** [comment]
fix mod seems to work for me on ~150 mod list, running sinytra as well

**Comment by Primo37** [comment]
still broken years later, why does this library mod even touch connected textures... the fix mod does fix it at least.
